packages/reports/src/Components/Form/CompositeCalendar/side-list.tsx

```diff
@@ -10,16 +10,16 @@ type TItem = {
 };
 
 type TSideList = {
-    from: number;
+    from: number | null;
     items: Array<TItem>;
     to: number;
 };
 
-const isActive = (from: number, to: number, flag: number) => {
+const isActive = (from: number | null, to: number, flag: number) => {
     if (flag === 0) {
         return toMoment().endOf('day').unix() === to && from === null;
     }
-    return Math.ceil(to / 86400) - Math.ceil(from / 86400) === flag;
+    return Math.ceil(to / 86400) - Math.ceil(Number(from) / 86400) === flag;
```

Comment:
For us it doesn't matter if `from` is `null`, because it will be treated as 0. Should not affect current production, but I can add a sanity checklist to the card if you think it will be need